lms/djangoapps/mobile_api/course_info/views.py

```diff
@@ -88,6 +88,15 @@ def list(self, request, course, *args, **kwargs):
                     course_id=course.id,
                     static_asset_path=course.static_asset_path
                 )
+                handouts_html = replace_jump_to_id_urls(
+                    handouts_html,
+                    course_id=course.id,
+                    jump_to_id_base_url="/courses/{}/jump_to_id/".format(course.id)
+                )
+                handouts_html = replace_course_urls(
+                    handouts_html,
+                    course_key=course.id
+                )
```

Comment:
I understand that this issue pre-exists, but..

I wonder if there's a good way to refactor this code so it is shared between LMS Web and Mobile API code.  Otherwise, if it's updated in LMS, developers won't know to update it here.